ably/realtime_conn.go

```diff
@@ -70,17 +70,29 @@ func (c *Conn) dial(proto string, u *url.URL) (proto.Conn, error) {
 // is non-nil.
 // If authorization succeeds, the returned Result value can be used to wait
 // until connection confirmation is received from a server.
-func (c *Conn) Connect() (Result, error) {
+func (c *Connection) Connect() (Result, error) {
 	return c.connect(true)
 }
 
+// ConnectV12 attempts to move the connection to the CONNECTED state, if it
+// can and if it isn't already.
+func (c *Connection) ConnectV12() {
```

Comment:
Is this still needed?

Reply:
Done at 7da5dbe129297ba6c5ca877256c538dada40f228.